### PR TITLE
Make raft clusters in tests come up quicker

### PR DESCRIPTION
**Pull request by ncabatoff**
…  via a smaller performance multiplier and thus quicker elections.

### CODE REVIEW

File: helper/testhelpers/teststorage/teststorage.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func MakeRaftBackend(t testing.T, coreIdx int, logger hclog.Logger) *vault.Physi
 	conf := map[string]string{
 		"path":                   raftDir,
 		"node_id":                nodeID,
-		"performance_multiplier": "8",
```
Comment (vishalnayak) [inline helper/testhelpers/teststorage/teststorage.go:98]: I didn't know that it could affect bootup times.

**Comment by calvn** [comment]
Are there any repercussions by lowering this value?

**Comment by ncabatoff** [comment]
> Are there any repercussions by lowering this value?

Brian points out that it may increase test flakiness, and indeed I see some raft race test failures that I don't recall seeing much before.  I may make this a CI-dependent (or CI-race-dependent) setting.

**Comment by ncabatoff** [comment]
Closing, see #10945 for a more complete & current version.

**Pull request closed without merging**